h/models/group.py

```diff
@@ -84,6 +83,9 @@ class Group(Base, mixins.Timestamps):
 
     scopes = sa.orm.relationship('GroupScope', backref='group', cascade='all, delete-orphan')
 
+    organization_id = sa.Column(sa.Integer, sa.ForeignKey('organization.id'))
```

Comment:
Is `organization_id` non-nullable by default? I think `organization_id` should be non-nullable / required, every group should have to belong to exactly one organization. But this is gonna mean more DB migrations will be needed:

* one to create a default Hypothesis organization
* and I think the migration that adds the `organization_id` column will need to create a nullable column, then set a column value for all preexisting groups, _then_ add the not-nullable constraint to the column

Reply:
@robertknight  @seanh https://github.com/hypothesis/h/pull/4886 is the db migration script. So we will do this adding organizations thing in the following order (feel free to correct this if I get something wrong or if you think these steps can be combined) 
1. Create an organization table, add the organization model, and add a nullable organization id to the group model.
1. Create a default hypothesis organization similar to the __world__ group only for organizations.
1. Migrate all groups to be under that default group and make sure creating a group always adds the group under the default organization if one is not specified. 
1. Change organization id so it is no longer nullable.
1. Migrate certain groups to be under a different organization (such as biopub). <--This will be the original story that started this mess.